.dependabot/config.yml

```diff
@@ -14,4 +14,6 @@ update_configs:
           dependency_name: "@jsii/*"
       - match:
           dependency_name: "codemaker"
-        
\ No newline at end of file
+      - match:
+          dependency_name: "semver"
```

Comment:
How do we remind ourselves to remove this? Can we keep a ticket open in our internal queue?